.env

```diff
@@ -1,14 +1,21 @@
+####################################################################################################
+# N.B. This file is intended as a placeholder. Never put any real values in this file and commit.
+####################################################################################################
+
+AZURE_ASSETS_STORAGE_CMS_ACCOUNT_CONTAINER='container'
+AZURE_ASSETS_STORAGE_CMS_ACCOUNT_KEY='key'
+AZURE_ASSETS_STORAGE_CMS_ACCOUNT_NAME='name'
+FARADAY_HOST='localhost:5000'
+FARADAY_X_FORWARDED_PROTO='http'
 FRONTEND_URL="localhost:5000"
 GA_API_EMAIL_ADDRESS="google.service.account@example.com"
 GA_PRIVATE_KEY_PATH="path/to/private.key"
+MAILJET_DEFAULT_FROM_ADDRESS=notmonitored@notify.moneyadviceservice.org.uk
+MAS_CMS_URL="http://localhost:3000/"
```

Comment:
Another vain attempt to organise the universe :laughing: What do you mean @Tr4pSt3R? My code is usually fairly intentional :stuck_out_tongue_winking_eye: